src/lemlib/devices/encoder/motor.cpp

```diff
@@ -12,37 +13,39 @@ namespace lemlib {
  * a reference, due to limitations in PROS 3. This is fixed in PROS 4, but
  * we have to deal with this for now.
  */
-MotorEncoder::MotorEncoder(std::shared_ptr<pros::MotorGroup> motors, float rpm)
+MotorEncoder::MotorEncoder(std::shared_ptr<pros::MotorGroup> motors, AngularVelocity speed)
     : motors(std::move(motors)),
-      rpm(rpm) {}
+      speed(rpm) {}
```

Comment:
```suggestion
      speed(speed) {}
```
I don't know how this compiled